src/test/java/com/epam/testsystem/github/dao/ApprovementDaoTest.java

```diff
@@ -0,0 +1,102 @@
+package com.epam.testsystem.github.dao;
+
+import com.epam.testsystem.github.TestUtil;
+import com.epam.testsystem.github.enums.ApprovementStatus;
+import com.epam.testsystem.github.model.Approvement;
+import com.epam.testsystem.github.model.Repo;
+import com.epam.testsystem.github.model.Task;
+import com.epam.testsystem.github.model.User;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.time.LocalDateTime;
+
+import static com.epam.testsystem.github.EnvironmentConstant.SPRING_PROFILE_TEST;
+import static org.assertj.core.api.Assertions.assertThat;
+
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+@ActiveProfiles(SPRING_PROFILE_TEST)
+public class ApprovementDaoTest {
+    @Autowired
+    private ApprovementDao approvementDao;
+
+    @Autowired
+    private TestUtil testUtil;
+
+    private User operator;
+    private Task task;
+
+    @Before
+    @Transactional
+    public void setUp() {
```

Comment:
It is very dangerous to mark setUp as transactional.
What do you expected more 
1) setUp will be an independent transaction and after the end of it, all information will be revert.
2) setUp will be an independent transaction but it will be never reverted and information from here can affect to other tests
3) or setting up and test will be in one transaction and everything will ok.

I do not know exactly. But i think it is not 3 way.